### PR TITLE
2 commits

### PR DESCRIPTION
**Pull request by adamtheschmidt**
slight clean of views/layout.jade as per your request

trims whitespace in register from front and back of all strings

### CODE REVIEW

File: views/login.jade [hunk 1]
```diff
@@ -21,9 +21,9 @@ block content
                     if message
                         div(style= "text-align:center" class="alert alert-"+ type +"" role="alert")= message
                         //alert-success
-                        //alert alert-info"
-                        //alert alert-warning
-                        //alert alert-danger
+                        //alert-info"
+                        //alert-warning
+                        //alert-danger
```
Comment (andreirtaylor) [inline views/login.jade:26]: Maybe we should have some documentation here saying what this means. Perhaps make this say that these are the options for the flash screen

File: routes/users.js [hunk 2]
```diff
@@ -83,19 +83,28 @@ router.post('/register', function(req, res, next) {
...
     //make sure the passwords match
     if(!(password === passConf)){
-        res.send("Passwords do not match");
-        return;
+        return res.render('register', {
+            message:"Passwords do not match",
+            type: "danger"});
+    }
+    //check for valid email
+    var re = /^\S+@\S+$/
```
Comment (andreirtaylor) [inline routes/users.js]: I cant merge this with this regex in here. Please fix. See my old comment

**Comment by adamtheschmidt** [comment]
regex from https://github.com/andreirtaylor/BoardGameTracking/pull/12

my bad - my mockup matched erroniously

**Comment by andreirtaylor** [comment]
Looks good. Onward and upward soldier

**Pull request merged**